index.d.ts

```diff
@@ -0,0 +1,25 @@
+interface CodeReturn<T> {
+  width: number,
+  height: number,
+  data: T
+}
+
+type BufferRet = CodeReturn<Buffer>;
+type UintArrRet = CodeReturn<Buffer>;
```

Comment:
should this be 
```suggestion
type UintArrRet = CodeReturn<Uint8Array>;
```

?

Reply:
This 100% should. Darn my copy+paste 🤦‍♂️